deployer/generate/generate_aws_cluster.py

```diff
@@ -5,7 +5,7 @@
 import jinja2
 import typer
 
-from .cli_app import app
+from deployer.cli_app import app
```

Comment:
I see source code using relative imports and absolute imports. I think the source code should stay consistently with relative imports.

In this case you can do this instead I think.

```suggestion
from ..cli_app import app
```

The one exception I see to use absolute imports is for anything outside the root module, for example if there is a `jupyterhub` folder for the jupyterhub source code, and a `tests` folder for the tests. Then the tests in the tests folder should reference `from jupyterhub import <stuff>` while if code is writting from within the jupyterhub folder, it would relate to code like `from .generate import stuff` or `from ..cli_app import app` etc.

Reply:
Thanks @consideRatio ✨ ! I usually believe that being explicit tends to be better, though I don't always respect this rule. I actually used relative imports here and changed it before pushing it upstream for this reason.

My **very personal** preference is to use relative imports when in the same directory (or when prototyping something and want to be quick), and use absolute ones when going up the hierarchy for readability and clarity purposes. 

But I agree this yells inconsistency and I understand why using relatives everywhere has its advantages. I added a commit to switch to relative imports https://github.com/2i2c-org/infrastructure/pull/1937/commits/b3ac262096958e7489f2b2a2d8e698264153d74b (and also spotted an issue with the repo root path thanks to this)

Reply:
Oooooh!!!

I've seen so much relative imports in the jupyterhub ecosystem, which I've used as a guide on whats right, that I assumed it was the recommended practice. Its the other way around according to PEP8 - absolute imports are recommended, but not strictly enforced.

- PEP8: https://peps.python.org/pep-0008/#imports
  ![image](https://user-images.githubusercontent.com/3837114/203553567-f996e496-9922-425f-b1ec-1082389c6b40.png)
- A text about absolute vs relative imports: https://realpython.com/absolute-vs-relative-python-imports/#pros-and-cons-of-absolute-imports

Thanks for iterating with me on this @GeorgianaElena! Do feel free to adjust as you think is best and disregard my previous comments where I thought relative imports were recommended.